server/plugin/notifications.go

```diff
@@ -67,15 +67,18 @@ func (p *Plugin) handleAddedComment(ctx context.Context, dSrv google.DriveInterf
 }
 
 func (p *Plugin) handleDeletedComment(userID string, activity *driveactivity.DriveActivity, file *drive.File) {
-	urlToComment := activity.Targets[0].FileComment.LinkToDiscussion
-	message := fmt.Sprintf("A comment was deleted in %s %s", utils.GetInlineImage("Google failed:", file.IconLink), utils.GetHyperlink(file.Name, urlToComment))
+	urlToComment := file.Name
```

Comment:
**question:** if we can't resolve a `LinkToDiscussion`, we'll use `file.Name` instead. What's the difference between the two?

Reply:
`LinkToDiscussion` will link directly to the conversation this activity is referencing but `file.Name` will at least tell them what file this comment is from. Now that you've pointed this out though I think I'll also make this a direct link to the file as well.